llvm/test/MC/RISCV/Relocations/mc-dump.s

```diff
@@ -9,6 +9,7 @@
 # CHECK-NEXT:0 Data LinkerRelaxable Size:8 [97,00,00,00,e7,80,00,00]
 # CHECK-NEXT:  Fixup @0 Value:specifier(19,ext) Kind:4023
 # CHECK-NEXT:  Symbol @0 $x
+# CHECK-NEXT:8 Data Size:0 []
```

Comment:
Wasn't the idea of this patch sequence to cut down on zero-sized data fragments? Is there something we can do here to get rid of this one?

Reply:
#149030 will remove this empty FT_Data. The fragment mechanism requires a large overhaul and I need to split the changes into pieces. The state after this patch does introduce an extra empty fragment for this corner case of RISC-V.